manifests/argo-rollouts.yaml

```diff
@@ -0,0 +1,293 @@
+# This yaml was generated from this command: kustomize build "https://github.com/argoproj/argo-rollouts/manifests/cluster-install?ref=v0.10.2"
```

Comment:
Make move this to a manifests/Makefile command: `make argo` ?

Reply:
I can try to do that. But that also means in that make command we will need to build 2 yaml files:
- `manifests/argo-rollouts.yaml` for all resources of argorollouts except CRDs
- `manifests/crd/argo-rollouts.yaml` for CRDs of argorollouts

Is that correct?

Reply:
Yeh - If make/yq isn't up to the job it could be done in a hack/update.go as well

Reply:
Implemented